src/applications/personalization/profile/components/ProfileWrapper.jsx

```diff
@@ -66,6 +69,10 @@ const ProfileWrapper = ({
 
   return (
     <>
+      {showUpdatedHeader &&
+        isLOA3 &&
```

Comment:
why do we care about LOA3?

Reply:
Because of [these specs in `When to show the name tag`](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/2.0-redesign/frontend/documentation/nametag.md)

Reply:
😅 👍